README.md

```diff
@@ -1,16 +1,14 @@
 # ![MathType logo](resources/img/mathtype_logo.png) MathType Web Integrations
 
-<img src="resources/img/wiris_plugin_example.png" alt="Wiris MathType plugin example" width="40%"/>
+<img src="resources/img/wiris_plugin_example.png" alt="Wiris MathType plugin example" width="40%" style="min-width: 480px"/>
 
-**MathType Web** is the official open-source JavaScipt library for [MathType by Wiris], the leading formula editor and equation writer for CMS, LMS, and web applications.
+**MathType Web** is the official open-source JavaScript library for [MathType by Wiris], the leading formula editor and equation writer for CMS, LMS, and web applications.
 
-This repository contains the plugins for the most popular HTML editors, such as [CKEditor], [Froala], and [TinyMCE].
-They are available for download on [npm](https://www.npmjs.com/~wiris).
-We also provide easy-to-use examples showing how to integrate MathType in [Angular] and [React] projects, or even your own custom HTML editor in your website.
-Our main goal is to enable web developers to integrate MathType in their web projects on the widest range of technologies available.
-We also try to keep the library light, simple, and free from third-party dependencies.
+This repository contains the plugins source-code for the most popular HTML editors, such as [CKEditor], [Froala], and [TinyMCE]. They are available from the [npmjs](https://www.npmjs.com/~wiris) directory. It also provides a collection of working examples, including step-by-step information, on how to integrate MathType in vanilla HTML, [Angular], [React] projects, or even your own custom web editor.
```

Comment:
A couple ideas:

- Reorder the word "plugins" after "source code". On the first read I missed the word plugins and thought that you meant that we hosted code for CKEditor, etc.
- [npm branding](https://www.npmjs.com/policies/trademark) drops the "js".

```suggestion
This repository contains the source code of the plugins for the most popular HTML editors, such as [CKEditor], [Froala], and [TinyMCE]. They are available from the [npm](https://www.npmjs.com/~wiris) directory. It also provides a collection of working examples, including step-by-step information, on how to integrate MathType in vanilla HTML, [Angular], [React] projects, or even your own custom web editor.
```

Reply:
Accepted